docs/usage.md

```diff
@@ -253,7 +253,15 @@ The following matrices are displayed by the SOC Management Widget:
    Detailed view of the record
   ![](./media/module_view.png)
 
-  
-  
+
+**Troubleshooting**
+
+**Key Not Found** 

```

Comment:
What is this? error message, log message?
Suggestion: 
Error: Key Not Found
Symptom:: This error is shown to an user, if a key or multiple keys are not present in an input JSON
Remediation:
- Please verify the JSON format is correct as mentioned in the widget documentation.
- Please check browser console logs for which key is missing.